backend/src/models/UserManager.js

```diff
@@ -0,0 +1,23 @@
+const AbstractManager = require("./AbstractManager");
+
+class UserManager extends AbstractManager {
+  constructor() {
+    super({ table: "user" });
+  }
+
+  insert(user) {
+    return this.database.query(
+      `insert into ${this.table} (username, password, mail, isadmin) values (?, ?, ?, false)`,
```

Comment:
Don't insert `false` value here. You can use a `?` and add false 
 into the array of values. You can also add a default value in table creation.